kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -148,11 +148,28 @@ data:
   agent-settings: |-
     # prometheus scrape fluent bit settings for high scale
     # buffer size should be greater than or equal to chunk size else we set it to chunk size.
+    # settings scoped to prometheus sidecar container
     [agent_settings.prometheus_fbit_settings]
       tcp_listener_chunk_size = 10
       tcp_listener_buffer_size = 10
       tcp_listener_mem_buf_limit = 200
 
+    # prometheus scrape fluent bit settings for high scale
+    # buffer size should be greater than or equal to chunk size else we set it to chunk size.
+    # settings scoped to daemonset container
```

Comment:
ideally, config parameter should have units in it. I understand that, you are existing Nam convention then probably lets have comments and units of these config parameters are  in "MBs"

Reply:
i'll add in comment